Doc/library/shutil.rst

```diff
@@ -48,7 +48,7 @@ Directory and files operations
    be copied.
 
```

Comment:
You should update the "Platform-dependent efficient copy operations" section:
https://docs.python.org/dev/library/shutil.html#platform-dependent-efficient-copy-operations